### PR TITLE
ISPN-14484 Document server tracing and client propagation

### PR DESCRIPTION
**Pull request by fax4ever**
https://issues.redhat.com/browse/ISPN-14484

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+At the time of writing tracing is defined at global level (once for all caches) passing the following system properties to the Infinispan server:
+
+.tracing.properties
+[source,options="nowrap",subs=attributes+]
+----
+include::properties/tracing.properties[]
+----
+<1> Globally enables the tracing.
+<2> The OTLP (OpenTelemetry protocol) exporter is the recommended format to export tracing data.
+<3> Specify here service name. The default name is `Infinispan Server`.
```
Comment (dvagnero) [inline documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc:14]: @fax4ever what exactly is the default? `Infinispan Server`?

File: documentation/src/main/asciidoc/topics/proc_propagation_hotrod_client_tracing.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[id='propagation-hotrod-client-tracing']
+= Hot Rod client tracing propagation
+The HotRod client fully supports the OpenTelemetry tracing propagation from the client to the Infinispan server.
+
+It means that if the OpenTelemetry tracing is configured both on client VM and on the {brandname} server,
```
Comment (dvagnero) [inline documentation/src/main/asciidoc/topics/proc_propagation_hotrod_client_tracing.adoc:5]: How do you configure tracing on client VM? Should we include the step?

File: documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+<4> Specify here the OpenTelemetry collector endpoint
+====
+
+.Procedure
+
+* Use `-D<property-name>=<property-value>` arguments to set tracing system properties as required.
+
+For our case.
+
+* Pass the following system properties to {brandname} Server at startup:
```
Comment (dvagnero) [inline documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc:24]: When the user wants to update the configuration, do they need to stop the server?

**Review by dvagnero — COMMENTED** [review]
Hi @fax4ever few questions for you. 
You've done an amazing job. The docs look great! 
I have few changes locally that I'd like to push + some more information depending on your answers. Can you please take a look at my comments?

File: documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc (hunk 1 above)
Comment (fax4ever) [reply documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc:14]: @dvagnero yes it is

File: documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc (hunk 3 above)
Comment (fax4ever) [reply documentation/src/main/asciidoc/topics/proc_configuring_tracing.adoc:24]: yes at the moment the configuration is taken at server startup time... more improvements will come

File: documentation/src/main/asciidoc/topics/proc_propagation_hotrod_client_tracing.adoc (hunk 2 above)
Comment (fax4ever) [reply documentation/src/main/asciidoc/topics/proc_propagation_hotrod_client_tracing.adoc:5]: It depends on the client technology, not by us. For instance a Quarkus OpenTelemetry extension or a SpringBoot OpenTelemetry module can be added for instance.

**Comment by fax4ever** [comment]
Great job @dvagnero. Merged it!

**Pull request merged**